Resources/snippets/frontend/mollie/plugins.ini

```diff
@@ -37,6 +38,7 @@ CardNumberLabel="Kreditkartennummer"
 CardExpiryDateLabel="Ablaufdatum"
 CardVerificationCodeLabel="CVC"
 CreditCardTagLine="Zahlung gesichert und bereitgestellt von"
+BlockedDueRiskManagement="Diese Zahlungsart ist leider für diesen Kauf nicht verfügbar. Sie können eine andere Zahlungsart und wählen und Ihren Einkauf fortführen."
```

Comment:
Zahlungsart und wählen

delete the und here

Reply:
uuuu danke :D